### PR TITLE
Fix/amount of files

### PR DESCRIPTION
**Pull request by MatthijsSmets**
remove ace editor files

### CODE REVIEW

File: src/app/shared/components/display-table/display-table.component.html [hunk 1]
```diff
@@ -1,69 +1,105 @@
...
+      Message is empty string
+    </div>
+
+    <div *ngIf="report.stubbed" id="stubbedMessageLabel" class="btn btn-static  px-2">
+      Message is stubbed
+    </div>
+
+    <div *ngIf="null !== report.encoding" id="encodedMessageLabel" class="btn btn-static px-2">
+      Message of type {{report.messageClassName}} is encoded to string using {{report.encoding}}
+      <span *ngIf="this.report.showConverted">and displayed using Base64 decoded to byte array and byte array decoded to string using UTF-8 (toggle with Base64 button)</span>
```
Comment (Matthbo) [inline src/app/shared/components/display-table/display-table.component.html:26]: Does this even work with the keyword `this` in there?

File: src/app/shared/components/display-table/display-table.component.html [hunk 2]
```diff
@@ -1,69 +1,105 @@
...
+      <div class="value" [cdkCopyToClipboard]="report.storage">{{ report.storage}}</div>
+    </div>
+    <div class='table-row' *ngIf='report.xml && report.estimatedMemoryUsage'>
+      <div class="key">Estimated memory usage</div>
+      <div class="value" [cdkCopyToClipboard]="report.estimatedMemoryUsage">{{ report.estimatedMemoryUsage}}</div>
+    </div>
+    <div class='table-row' *ngIf='report.xml && report.correlationId'>
+      <div class="key">CorrelationId</div>
+      <div class="value" [cdkCopyToClipboard]="report.correlationId">{{ report.correlationId}}</div>
+    </div>
```
Comment (Matthbo) [inline src/app/shared/components/display-table/display-table.component.html:103]: Maybe an idea but cant you wrap this in an `ng-container` with an if / else check on `report.xml`, that way more readable and you dont have to do the check the same property on every condition?

File: src/app/shared/components/display-table/display-table.component.ts [hunk 3]
```diff
@@ -1,25 +1,10 @@
...
-  }
-
-  public _report: any = {};
-
-  constructor(private helperService: HelperService) {}
-
-  getCheckpointType(type: number) {
-    return this.helperService.getCheckpointType(type);
-  }
+  @Input() report!: any;
```
Comment (Matthbo) [inline src/app/shared/components/display-table/display-table.component.ts:9]: You have enough report types to choose from right? Or is it not that straight forward once again?

File: src/app/shared/pipes/checkpoint-type.pipe.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { HelperService } from '../services/helper.service';
+
+@Pipe({
+  name: 'checkpointType',
+  standalone: true,
```
Comment (Matthbo) [inline src/app/shared/pipes/checkpoint-type.pipe.ts:6]: Do you have an idea on if you want to make everything standalone at some point? Making this one standalone isnt bad but lets think about the future, maybe making everything standalone isnt a bad thing

File: src/app/debug/display/display.component.html [hunk 5]
```diff
@@ -3,10 +3,11 @@
...
       <div data-cy-debug-editor="buttons" class="row" id="displayButtons">
         <app-button data-cy-debug-editor="copy" [icon]="'fa fa-copy'" [title]="'Copy'" (click)="copyReport()"></app-button>
-        <app-button *ngIf="report.encoding === 'Base64'"
-                    [text]="'Base64'"
-                    [title]="'Convert to Base64'"
-                    (click)="changeEncoding($event)"></app-button>
+        <app-button
+          *ngIf="report.encoding === 'Base64'"
+          [text]="'Base64'"
+          [title]="'Convert to Base64'"
```
Comment (philipsens) [inline src/app/debug/display/display.component.html:9]: Binding is not needed here, is it not?

**Pull request closed without merging**